src/challenges/challengeOne/4.Form/Form.test.tsx

```diff
@@ -1,8 +1,61 @@
-// import React from "react";
-// import { render, screen } from "@testing-library/react";
-// import userEvent from "@testing-library/user-event";
-// import { Form } from "./Form";
-
-// describe("Form", () => {
-//   it("renders...", () => {});
-// });
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { Form, FormProps } from "./Form";
+
+const testFields: FormProps["fields"] = [
+  {
+    label: "Name",
+    type: "text",
+    name: "name",
+    placeholder: "Name",
+    value: "",
+    id: "name",
+  },
+  {
+    label: "Email",
+    type: "email",
+    name: "email",
+    placeholder: "Email",
+    value: "",
+    id: "email",
+  },
+];
+
+const onSubmitMock = jest.fn();
+
+describe("Form", () => {
+  const renderComponent = (
+    fields: FormProps["fields"],
+    onSubmit: FormProps["onSubmit"],
+  ) => {
+    render(<Form fields={fields} onSubmit={onSubmit} />);
+  };
+
+  it("renders without error", () => {
+    renderComponent(testFields, () => {});
+
+    const nameLabel = screen.getByLabelText("Name");
+    const emailLabel = screen.getByLabelText("Email");
+
+    expect(nameLabel).toBeInTheDocument();
+    expect(emailLabel).toBeInTheDocument();
+  });
+
+  it("calls onSubmit with correct values on submit", async () => {
```

Comment:
You should really to test that you can fill in the inputs before you test that you can submit